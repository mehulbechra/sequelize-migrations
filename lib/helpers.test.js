const helpers = require('./helpers');

describe('test for revisedBookData', async () => {
  it('output price to be revised based on year', async () => {
    const bookData = [
      {
        id: 1,
        price: 10,
        year: '2000',
      },
      {
        id: 2,
        price: 10,
        year: '2010',
      },
    ];
    expect(helpers.revisedBookData(bookData)).toMatchSnapshot();
  });
});
