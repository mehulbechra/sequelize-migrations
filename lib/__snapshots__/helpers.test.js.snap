// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test for revisedBookData output price to be revised based on year 1`] = `
Array [
  Object {
    "id": 1,
    "price": 5,
    "year": "2000",
  },
  Object {
    "id": 2,
    "price": 8,
    "year": "2010",
  },
]
`;
